docs/using_lagoon/active_standby.md

```diff
@@ -112,4 +119,39 @@ mutation updateProject {
     standbyAlias
   }
 }
+```
+
+## Notes
+
+When the active/standby trigger has been executed, the `productionEnvironment` and `standbyProductionEnvironments` will switch within the Lagoon API. Both environments are still classed as `production` environment types. We use the `productionEnvironment` to determine which one is labelled as `active`.
```

Comment:
Maybe we should link to the Environment Types page and say something like "for more information on the difference between production and development environments, read the documentation on environment types." I'm worried this could be confusing otherwise.

Reply:
Ahh good idea :D I will make some changes.